---
bool status
geometry_msgs/Vector3[] goal_poses